spec/datetimeformat.html

```diff
@@ -57,11 +53,11 @@ <h1>InitializeDateTimeFormat ( _dateTimeFormat_, _locales_, _options_ )</h1>
         1. Set _opt_.[[localeMatcher]] to _matcher_.
         1. Let _calendar_ be ? GetOption(_options_, *"calendar"*, ~string~, ~empty~, *undefined*).
         1. If _calendar_ is not *undefined*, then
-          1. If _calendar_ does not match the Unicode Locale Identifier `type` nonterminal, throw a *RangeError* exception.
+          1. If _calendar_ cannot be matched by the <code>type</code> language tag nonterminal, throw a *RangeError* exception.
         1. Set _opt_.[[ca]] to _calendar_.
```

Comment:
Question: Should a calendar value like "foo_bar" be rejected here (and analogously for similar steps in other algorithms)? It is a well-formed `type` according to the _loose_ grammar rules of <a href="https://unicode.org/reports/tr35/#Unicode_Language_and_Locale_Identifiers">Unicode Technical Standard #35 Part 1 Core, Section 3 Unicode Language and Locale Identifiers</a> that support `_` as a separator, but is incompatible with BCP 47 and would be rejected by ECMA-402 IsStructurallyValidLanguageTag for using such ["backwards compatibility" syntax](https://unicode.org/reports/tr35/#BCP_47_Conformance). And regarding web reality, ChakraCore seems to be the only implementation that accepts it (as opposed to all Intl-supporting implementations accepting the strictly well-formed "foo-bar").